pkg/resource/deploy/plan_executor.go

```diff
@@ -194,6 +194,10 @@ func (pe *planExecutor) Execute(callerCtx context.Context, opts Options, preview
 	pe.stepExec.WaitForCompletion()
 	logging.V(4).Infof("planExecutor.Execute(...): step executor has completed")
 
+	if pe.stepGen.hasPolicyViolations {
```

Comment:
Sorry, I was suggesting moving this down to line 206:

```
	// Figure out if execution failed and why. Step generation and execution errors trump cancellation.
	if res != nil || pe.stepExec.Errored() || pe.stepGen.hasPolicyViolations {
```

Reply:
@pgavlin Done. I actually like _not_ emitting the "x failed" message, but understand the desire to keep it. Anything else?